source/translator.d

```diff
@@ -0,0 +1,34 @@
+private string[string] defaultLanguage;
+static this()
+{
+	defaultLanguage = [
+		"run": "Run",
+		"reset": "Reset",
+		"keyboard_shortcuts": "Keyboard Shortcuts",
+	];
+}
+
+/**
+Allows per-language overwrites of UI text
+*/
+class Translator
+{
+	string[string] translations;
+
+	// in case no translation is found
+	this() {}
+
+	this(string[string] translations)
+	{
+		this.translations = translations;
+	}
+
+	// uses default language as fallback
+	string opIndex(string key) const
```

Comment:
I tested this a while ago and I didn't manage to find a difference, so I'd say there's no need to worry for now.